examples/JavaScript/WebDemoFeatureFlag/azure-app-configuration.js

```diff
@@ -0,0 +1,31 @@
+const { AppConfigurationClient } = require("@azure/app-configuration");
+
+if (!process.env.production) {
+  require("dotenv").config();
+  console.log("debug environment");
+}
+
+const connectionString = process.env.APPCONFIG_CONNECTION_STRING;
+console.log(connectionString)

```

Comment:
Please don't log the connection string. Not a good practice to log secrets.

Reply:
@zhenlan Removed, please check the PR now